.github/workflows/smoke.yml

```diff
@@ -0,0 +1,110 @@
+name: smoke
```

Comment:
nothing changed in here but a few comments and the `name` values for all three jobs